app/models/oidc/trusted_publisher/github_action.rb

```diff
@@ -41,7 +41,6 @@ def self.permitted_attributes
 
   def self.build_trusted_publisher(params)
     params = params.reverse_merge(repository_owner_id: nil, repository_name: nil, workflow_filename: nil, environment: nil)
-    params.delete(:environment) if params[:environment].blank?
```

Comment:
I wonder of this should be `params[:environment] = nil if params[:environment].blank?` to coerce the empty string to nil (or alternatively, `params[:environment] = params[:environment].presence`)

Reply:
This makes sense to me. It's not necessarily obvious if a form has an empty string, so this makes for an improved user experience as well 😄